src/libs/actions/Report.ts

```diff
@@ -150,6 +150,8 @@ Onyx.connect({
         }
         const reportID = CollectionUtils.extractCollectionItemID(key);
         currentReportData[reportID] = report;
+        // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
can we avoid suppressing this lint rule?

Reply:
@roryabraham  This is rather complex. We have `handleReportChanged` declared way down in the file and it depends on `reconnect` which is declared above `handleReportChanged`. So to avoid this lint rule, we will have to bring these two functions just above the `Onyx.connect` call. 

Which will look bad in terms of readability because as of now we have all `Onyx.connect` calls in one place and below that we have functions.

Reply:
ok, no worries then